armory/datasets/preprocessing.py

```diff
@@ -57,6 +58,54 @@ def carla_over_obj_det_dev(element, modality="rgb"):
     )
 
 
+def carla_video_tracking_preprocess(element, max_frames, split):
```

Comment:
I'd suggest separating the video preprocessing and the label preprocessing into separate functions to make things more modular

Reply:
See https://github.com/twosixlabs/armory/pull/1836/files/f07ae034e7a4415bf0bfeb3c543133bdf4e7b194#r1055907094.